chore(deps): py bump docker ^7.1.0, requests ^2.32.2 (#115)
kubeflow/model-registry: CONTRIBUTING.md

Several options of docker engines are available for Mac.
Having Docker installed is also helpful for Testcontainers.

### Podman

[Podman](https://podman.io/) also supports Rosetta (Apple specific) emulation which is handy since the Google MLMD project dependency is x86 specific.

> We recommend setting up the Podman machine with root priviledges,

Author: ```suggestion
We recommend setting up the Podman machine with root privileges,
```

Several options of docker engines are available for Mac.
Having Docker installed is also helpful for Testcontainers.

### Podman

Podman also supports Rosetta (Apple specific) emulation which is handy since the Google MLMD project dependency is x86 specific.
You can install Podman from the Podman [website](https://podman.io/).

> We recommend setting up the Podman machine with root priviledges,
> and setting the environment variable
> 
> ```sh
> export TESTCONTAINERS_RYUK_PRIVILEGED=true
> ```
> 
> when running TestContainer-based Model Registry Python tests (for more information, see [here](https://pypi.org/project/testcontainers/#:~:text=TESTCONTAINERS_RYUK_PRIVILEGED)).

Reviewer: IIRC by exporting the env variable we're already following the recommended usage, so I'd rewrite the recommendation, mainly taking out the "and" (as it sounds like there is something additional to be done when using the Podman machine in that sense). Also, nit (typo).

```suggestion
We recommend using a [privileged Podman machine](https://pypi.org/project/testcontainers/#:~:text=TESTCONTAINERS_RYUK_PRIVILEGED)) when running TestContainer-based Model Registry Python tests.
You can do that by setting the following environment variable

```sh
export TESTCONTAINERS_RYUK_PRIVILEGED=true
```

```

Author: In my understanding they're not the same thing, as this proposed correction would make it look like (since the "podman machine" is now linking to Ryuk container privileges).

In my understanding, one thing is the emulation used to run containers in general on macOsx, for podman that is the "podman machine"; another thing is the Ryuk container, used by TestContainers as a ~"garbage collector" which need to opt-in to this privileges.

I have podman machine as root by default,
but if I miss to specify the environment variable for ryuk specifically it won't work for Testcontainers.

In conclusion, to me it's indeed an "and".

Reviewer: thanks for the heads up!
